evil-colemak-basics.el

```diff
@@ -118,6 +128,14 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
           "j" 'evil-snipe-j
           "J" 'evil-snipe-J))
        (t (user-error "Invalid evil-colemak-basics-char-jump-commands configuration"))))
+    (when (eq evil-colemak-basics-layout-mod 'mod-dh)
+      (evil-define-key '(motion normal visual) keymap
+        "m" 'evil-backward-char)
+      (evil-define-key 'normal keymap
+	"h" 'evil-set-marker)
+      (evil-define-key 'motion keymap
+	"H" 'evil-window-top
+	"M" 'evil-window-middle))
```

Comment:
Sorry if I miss something, but are "H" and "M" not already mapped to those commands in Evil by default?
If it's really needed, perhaps we should write some comment here.

Reply:
you may be right, these seem unaltered

Reply:
fixed in db01118a76112c61b7617aa44aa8c438d8f1871f